install/linux/usr/bin/odemis-select-mic-start

```diff
@@ -46,17 +46,38 @@ if [ $status == 0 ] || [ $status == 3 ] ; then
 fi
 
 mic_list=()
+error_filenames=()
+mkdir $HOME/.local/share/odemis/log 2>/dev/null
+parse_logfile=$HOME/.local/share/odemis/log/odemis-select-mic-start-parse.log
+> $parse_logfile
+
 for fn in "${args[@]}"; do
+    # add a new header to the logfile
+    printf "$(date) Parsing file: $fn\n" >> $parse_logfile
     # extract the microscope name by using this small Python script
-    # note: does not check for errors when parsing
-    mic_name="$(python3 -c "import sys; import yaml; from odemis.odemisd import modelgen; f = open('$fn'); d = yaml.load(f, modelgen.SafeLoader); print(next(n for n, d in d.items() if d.get('class') == 'Microscope'))" 2>/dev/null)"
+    mic_name="$(python3 -c "import sys; import yaml; from odemis.odemisd import modelgen; f = open('$fn'); d = yaml.load(f, modelgen.SafeLoader); print(next(n for n, d in d.items() if d.get('class') == 'Microscope'))" 2>>$parse_logfile)"
```

Comment:
This is a hard one to parse. Since it's python with clear separate commands, can we multi-line it for better readability?

Reply:
I see it is more symptomatic in this file, to have long single lines for bash commands. For better readability and git diff tracking, I would prefer multi lines.